src/compat.h

```diff
@@ -10,6 +10,16 @@
 #include <config/bitcoin-config.h>
 #endif
 
+#include <type_traits>
+
+// GCC 4.8 is missing some C++11 type_traits,
+// https://www.gnu.org/software/gcc/gcc-5/changes.html
+#if defined(__GNUC__) && __GNUC__ < 5
+#define IS_TRIVIALLY_CONSTRUCTIBLE std::is_trivial
+#else
+#define IS_TRIVIALLY_CONSTRUCTIBLE std::is_trivially_constructible
+#endif
```

Comment:
Is this redundant with the same in `prevector.h`?

Reply:
Good catch, pushing a rebase that fixes this.